TestON/drivers/common/cli/emulator/remotemininetdriver.py

```diff
@@ -376,9 +376,20 @@ def disconnect( self ):
         if self.handle:
             # Close the ssh connection
             self.handle.sendline( "" )
-            self.handle.expect( "\$" )
-            self.handle.sendline( "exit" )
-            self.handle.expect( "closed" )
+            #self.handle.expect( "\$" )
+            i = self.handle.expect( [ '\$', 'mininet>', pexpect.TIMEOUT ],
```

Comment:
"""
    RemoteMininetCliDriver is the basic driver which will handle the Mininet
    functions. The main different between this and the MininetCliDriver is that
    this one does not build the mininet. It assumes that there is already a
    mininet running on the target.
    """
If you need to run something on lincoe you should using lincoe driver correct? We mentioned above in remotemininet driver that it should not be used to build mininet.